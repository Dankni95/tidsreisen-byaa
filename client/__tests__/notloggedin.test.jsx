import React from "react";
import { MemoryRouter } from "react-router-dom";
import { NotLoggedIn } from "../src/components/NotLoggedIn.jsx";
import renderer from 'react-test-renderer';

describe("NotLoggedIn", () => {
    it("shows snapshot", async () => {
        const component = renderer.create(
            <MemoryRouter>
                <NotLoggedIn />
            </MemoryRouter>
        );
        let tree = component.toJSON()
        expect(tree).toMatchSnapshot();
    });
});