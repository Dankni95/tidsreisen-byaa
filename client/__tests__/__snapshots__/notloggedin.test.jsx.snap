// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotLoggedIn shows snapshot 1`] = `null`;
